.github/dependabot.yml

```
@@ -32,11 +32,3 @@ updates:
     # # Managed by cisagov/cool-iam-user-group-manager-iam
```

Not sure what happened here:
```suggestion
    # # Managed by cisagov/skeleton-tf-module
```

---

Yeah, not sure what happened there either, but I cleaned it up in https://github.com/cisagov/cool-iam-user-group-manager-iam/pull/7/commits/069b354cf451d35936103a35ddcaad9f76cf49fc.

---

@dav3r I fixed this in https://github.com/cisagov/gh-skeleton/pull/30.